lib/linguist/languages.yml

```diff
@@ -163,13 +163,17 @@ Ada:
   type: programming
   color: "#02f88c"
   extensions:
-  - ".adb"
```

Comment:
Why did you change this? Is `.adb` not the primary extension?

Reply:
`.adb` is still in the list. Extensions were re-ordered to be alphabetical.

Reply:
@rkoeninger Note the part in @pchaigno's comment about **primary extension**. Linguist makes an exception to the alphabetisation rule in that the "primary" file extension (the one most commonly used or associated with a language) comes first, irrespective of anything else.

It's slightly confusing for new contributors, I'll admit. Please make sure you've read not just [`CONTRIBUTING.md`](https://github.com/github/linguist/blob/master/CONTRIBUTING.md), but also the field documentation at the top of the `languages.yml` file. You'll notice that the `extensions` field [has an explanation](https://github.com/github/linguist/blob/4cd558c/lib/linguist/languages.yml#L12-L14) about the "primary extension" matter:

> **`extensions`** - An Array of associated extensions (the first one is considered the primary extension, the others should be listed alphabetically)

Reply:
Oh, sorry, I didn't see that. I'll push a correction.